### PR TITLE
Feature branch for  website

### PR DESCRIPTION
**Pull request by arslanbisharat**
### resturant-page
**I used JavaScript alone to generate the entire contents of the website!**
> Readme has enough information to run this repo
>It has three main section About, menu and location
> location is  not workng right  now.

### CODE REVIEW

File: src/food.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export default function foodMenu() {
+
+    const products = ['HalwaPuri', 'Biryani', 'Chicken Karahi', 'Lassi', 'Drinks', 'Desserts']
+    const productsUrls = ['images/HalwaPuri.jpeg', 'images/Biryani.jpeg', 'images/Chicken.jpeg', 'images/Lassi.jpg', 'images/Drinks.jpg', 'images/dessert.jpeg']
```
Comment (maelfosso) [inline src/food.js]: Screenshot at 2020-07-05 14:04:17
This error is due to the wrong name of images. Kindly fix it.

File: src/about_us.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            aboutText.style.padding = '0 20%'
+            aboutContainer.appendChild(aboutText)
+            
+            const img = document.createElement('img')
+            img.src = 'images/kitchen.jpeg'
+            img.classList.add('about-img')
+            aboutContainer.appendChild(img)
+        
+        content.appendChild(aboutContainer)
+}
```
Comment (maelfosso) [inline src/about_us.js]: Make your code more readable.
Indent it correctly. Update also the others files, please.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for this well done job.
Before merging, please read the comments below and submit a new code review request
Also, include `.vscode` into `.gitignore`.

Thanks

Mael FOOSO

**Review by fatymahmed — APPROVED** [review]
## Hi @arslanbisharat,
Great job making the requested changes 🎉🎉
Please make the following change before merging.
- Make sure to update `dist/main.js` with the latest changes you made

[OPTIONAL]
- Set up linter, [here's](https://github.com/microverseinc/linters-config/tree/master/javascript) how to do so
## Status: Approved ✔️

**Pull request merged**